src/vulkan/device.cc

```diff
@@ -390,16 +393,21 @@ Result Device::Initialize(
     PFN_vkGetInstanceProcAddr getInstanceProcAddr,
     Delegate* delegate,
     const std::vector<std::string>& required_features,
-    const std::vector<std::string>& required_extensions,
+    const std::vector<std::string>& required_instance_extensions,
+    const std::vector<std::string>& required_device_extensions,
     const VkPhysicalDeviceFeatures& available_features,
     const VkPhysicalDeviceFeatures2KHR& available_features2,
     const std::vector<std::string>& available_extensions) {
   Result r = LoadVulkanPointers(getInstanceProcAddr, delegate);
   if (!r.IsSuccess())
     return r;
 
-  bool use_physical_device_features_2 = false;
-  for (auto& ext : required_extensions) {
+  const bool isVulkan11Plus =
+      (VK_VERSION_MAJOR(physical_device_properties_.apiVersion) >= 1 &&
+       VK_VERSION_MINOR(physical_device_properties_.apiVersion) >= 1);
+
+  bool use_physical_device_features_2 = isVulkan11Plus;
```

Comment:
This change seems unrelated to the change in progress, can it be moved to its own change.